portfolio/src/main/webapp/script.js

```diff
@@ -42,3 +42,22 @@ function showSlides(newIndex) {
   slides[slideIndex].style.display = "block";
   dots[slideIndex].className += " active";
 }
+
+/** Displays list of comments returned by the server.*/
+function showComments() {
+  fetch('/comments').then(response => response.json()).then(comments => {
+    const commentsElement = document.getElementById('text-container');
+    commentsElement.innerHTML = '';
+    for(let i = 0; i < comments.length; i++) {
+      commentsElement.appendChild(
+      createListElement(comments[i]));
```

Comment:
Indentation is not correct here. Actually I think this still fits into the line before.